triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -64,7 +112,19 @@ object ServiceConfig {
         c.copy(commandTtl = Duration.ofSeconds(t))
       }
       .text("TTL in seconds used for commands emitted by the trigger. Defaults to 30s.")
+
+    opt[Map[String, String]]("jdbc")
```

Comment:
We now have `--jdbc` for the trigger service, `--query-store-jdbc-config` for the JSON API and `--sql-backend-jdbcurl` for sandbox the latter even has a different format :crying_cat_face:

Reply:
Good point, I could use the sandbox format and argument name, by including the username and password in the url string. I'll think about tweaking this in a later PR if that's ok.

Reply:
Happy to do it separately, I’d just like to not have to remember 3 different ways of specifying a jdbc url in the long term :slightly_smiling_face: